src/axom/core/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ set(core_headers
     utilities/BitUtilities.hpp
     utilities/CommandLineUtilities.hpp
     utilities/FileUtilities.hpp
+    utilities/NumericArray.hpp
```

Comment:
NumericArray looks like StackArray plus some arithmetic operators. All of the other arrays are in the root level in the core directory. It doesn't match to have NumericArray one level down in utilities.

Reply:
I agree that this should be in the same folder as the other array classes

Reply:
Ok.